docker/ubuntu/Dockerfile

```diff
@@ -19,6 +19,10 @@ update-alternatives --install /usr/bin/gcc gcc /usr/bin/gcc-8 60 --slave /usr/bi
 update-alternatives --config gcc; \
 apt-get install -y wget openssl libssl-dev git make
 
+# will put repo.hops.works
+RUN wget http://archive.ubuntu.com/ubuntu/pool/main/o/openssl/libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+RUN -i libssl1.1_1.1.1f-1ubuntu2_amd64.deb
+
```

Comment:
this won't work and i don't think it would be needed for building ePipe

Reply:
Don't you run epipe in a docker container sometimes?